CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 See [here](https://github.com/charliermarsh/ruff/releases) for the Ruff release notes.
 
+## 2024.17.0
+
+This pre-release introduces support for the new, experimental Ruff language server (`ruff server`). It also upgrades the bundled Ruff version to `v0.3.3`.
+
+`ruff server` is still missing a lot of tests and stability guarantees - this pre-release is mostly for early testing. The server can be enabled and disabled with the `experimentalServer` setting. A restart is required after changing this setting.
```

Comment:
Maybe: "`ruff server` is still missing some core functionality and stability guarantees - the pre-release is intended for early testing "?